ci/stack-orchestration/manual-stacks/txma/template.yml

```diff
@@ -81,6 +81,46 @@ Resources:
         - Key: 'ProducerType'
           Value: 'SQS'
 
+  TxMAAuditEventQueueMessageAgeCloudWatchAlarm:
+    Condition: TxMAIntegrationEnabled
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: txma-audit-event-queue-message-age-alarm
+      AlarmDescription: !Sub "${AWS::StackName}-AuditEventQueue contains message(s) older than 24 hours which means TxMA is failing to ingest events. ACCOUNT: di-orchestration-${Environment}."
+      Namespace: AWS/SQS
+      MetricName: ApproximateAgeOfOldestMessage
+      Dimensions:
+        - Name: QueueName
+          Value: !GetAtt TxMASQSProducerAuditEventQueue.QueueName
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 86400 # 24 hours in seconds
+      EvaluationPeriods: 1
+      Period: 60
+      Statistic: Maximum
+      AlarmActions:
+        - !ImportValue
```

Comment:
This syntax looks slightly odd. Willing to believe it though if you're sure

Reply:
Agreed, was the only way I could find that would pass validation. Will take another quick look before merging